packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -66,10 +68,15 @@ export default class Bridge extends ContractBase {
     }
     this.db = getDbSet(this.tokenSymbol)
     const bridgeDeployedBlockNumber = config.tokens[this.tokenSymbol]?.[this.chainSlug]?.bridgeDeployedBlockNumber
+    const l1CanonicalTokenAddress = config.tokens[this.tokenSymbol]?.[Chain.Ethereum]?.l1CanonicalToken
```

Comment:
Out of curiouslity, why is `stateUpdateAddress` passed in via constructor config argument but `l1CanonicalTokenAddress` is being read directly from global config here? Should stateUpdateAddress be read the same way as well, so it's less code?

Reply:
Does `stateUpdateAddress` being read from global config imply that it lives in the `addresses` package?

Reply:
The global config contains things from addresses package but also things outside of it (eg like the min/max limits)

Reply:
Resolved with f7551d37755cb17e805c105691b41a8520890963